Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14 AS builder
+WORKDIR /build
+COPY . .
+RUN mkdir -p bin && \
+    go build -o bin ./cmd/cluster-olm-operator
```

Comment:
Any reason not to use `make build` here?

Reply:
No, I added make build after I wrote this 👍